src/components/details/GitOpsDetailsPageHeading.tsx

```diff
@@ -0,0 +1,58 @@
+import './GitOpsDetailsPageHeading.scss';
```

Comment:
I thought if we lint/sort this, this one will go at the bottom?

Reply:
I used an extension called TypeScript Import Sorter, and it says it default follows ESLint sort-imports rules. So I didn't move it to the bottom. Can we leave this issue in the Linting ticket?